mantis-network/src/main/java/io/reactivex/mantis/network/push/HashFunctions.java

```diff
@@ -37,6 +38,26 @@ public long computeHash(byte[] keyBytes) {
         };
     }
 
+    public static HashFunction ketamaExtended() {
```

Comment:
do we need to keep this if the default impl switched to use xxh3?

Reply:
We do not. I'll make the change.